vidarrtest-regression.json.in

```diff
@@ -4,7 +4,7 @@
             "mutect2.filter.filterExtraArgs": null,
             "mutect2.reference": "hg19",
             "mutect2.gatk": "gatk/4.1.6.0",
-            "mutect2.outputFileNamePrefix": "ESBCC_0005_Tumor_Matched_Normal_EX",
```

Comment:
I would check all outputFileNamePrefix in this file and revert everything back to original values.